### DISCUSSION TITLE
Can't install plugins in v5.0.4

### DISCUSSION THREAD

**Discussion by z7924723** [docker-library/redmine#284] [General]
Hi all,

I have a question about installing plugins.

I use docker to deploy redmine v5.0.4 and want to install the "redmine_code_review" plugin.
After executing the below commands inside the container, no plugin install.
```
bundle install --without development test
bundle exec rake redmine:plugins:migrate name=redmine_code_review version=0 rails_env=production
```

But, these commands can successfully install the "redmine_code_review" plugin under v4.2.9.
Does someone have the same problem?

**Comment by Romich777** [comment]
Remove `version=0`, AFAIK this used to remove plugin related data from database, created by migration itself
